fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -116,6 +153,22 @@ class EqualizedOdds(ConditionalSelectionRate):
 
     .. math::
        P[h(X) = y' | A = a, Y = y] = P[h(X) = y' | Y = y] \; \forall a, y, y'
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    events corresponding to the unique values of the `Y` array.
+
+    The `prob_event` :class:`pandas:pandas.DataFrame` will record the
+    fraction of the samples corresponding to each unique value in
+    the `Y` array.
+
+    The `index` MultiIndex will have a number of entries equal to
+    the number of unique values for the sensitive feature, multiplied by
+    the number of unique values of the `Y` array, multiplied by two (for
+    the positive and negative Lagrange multipliers).
+
```

Comment:
similar edit as above for "positive and negative Lagrange multipliers"

Reply:
Done